test/e2e/util/utils_test.go

```diff
@@ -0,0 +1,81 @@
+package util
+
+import (
+	"testing"
+
+	"k8s.io/klog/v2"
+)
+
+func TestParseRepoTag(t *testing.T) {
+	cases := []struct {
+		input     string
+		expected  RepoTag
+		expectErr bool
+	}{
+		{
+			input: "ghcr.io/repo/one/two:three",
+			expected: RepoTag{
+				Repo: "ghcr.io/repo/one/two",
+				Tag:  "three",
+			},
+			expectErr: false,
+		},
+		{
+			input: "ghcr.io/one:two",
+			expected: RepoTag{
+				Repo: "ghcr.io/one",
+				Tag:  "two",
+			},
+			expectErr: false,
+		},
+		{
+			input: "eraser:e2e-test",
+			expected: RepoTag{
+				Repo: "eraser",
+				Tag:  "e2e-test",
+			},
+			expectErr: false,
+		},
+		{
+			input: "",
+			expected: RepoTag{
+				Repo: "",
+				Tag:  "",
+			},
+			expectErr: false,
+		},
+		{
+			input: ":",
+			expected: RepoTag{
+				Repo: "",
+				Tag:  "",
+			},
+			expectErr: false,
+		},
+		{
+			input: "/",
+			expected: RepoTag{
+				Repo: "",
+				Tag:  "",
+			},
+			expectErr: true,
+		},
```

Comment:
can we add a test case with a valid and invalid digest too?

Reply:
done